### DISCUSSION TITLE
充值成功，兑换激活失败

### DISCUSSION THREAD

**Discussion by AveryLRG** [BurnHair/docs#174] [技术支持]
通过支付宝充值3美元，然后支付成功返回网页报错，我找支付宝订单号，找到对应激活码，去激活时发现已被激活，麻烦管理帮忙看看什么原因，谢谢

**Comment by BennyThink** [comment]
是我弄出来了bug。你看下邮件，我给你回复了

**Reply by AveryLRG** [reply]
看到了，解决了，谢谢
